tasks/util/watchified_bundle.js

```diff
@@ -17,8 +17,8 @@ var common = require('./common');
  * @param {function} onFirstBundleCallback executed when first bundle is completed
  *
  */
-module.exports = function makeWatchifiedBundle(onFirstBundleCallback) {
-    var b = browserify(constants.pathToPlotlyIndex, {
+module.exports = function makeWatchifiedBundle(strict, onFirstBundleCallback) {
+    var b = browserify(strict ? constants.pathToPlotlyStrict : constants.pathToPlotlyIndex, {
```

Comment:
Non-blocking, but do we actually want the strict dashboard to use the strict bundle? I'd kind of think we want the full bundle there, or at least have that capability, in order to test which of the full set of traces and features work.

Reply:
That could become an option to 'npm start'.